ci/long_running_tests/config.yaml

```diff
@@ -0,0 +1,65 @@
+cluster_name: default
+min_workers: 0
+max_workers: 0
+target_utilization_fraction: 0.8
+idle_timeout_minutes: 5
+
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-west-2
+    availability_zone: us-west-2a
+auth:
+    ssh_user: ubuntu
+
+head_node:
+    InstanceType: m5.12xlarge
```

Comment:
Is it possible to use a smaller node by default? It would be nice if anyone could run this without worrying too much about costs (also, I think you need to query AWS to increase 12xlarge instance limits?)

Reply:
These are now `m5.2xlarge` which cost $0.384 per hour.

Reply:
It'd be nice to make it cheaper... if we're running this continuously say with 20 workloads, that's $180/day.